### PR TITLE
Feature/rickwoltheus/participants registrations overview query

### PR DESCRIPTION
**Pull request by RickWoltheus**
this pr adds the overview query to the participant registration view

### CODE REVIEW

File: client/src/graphql/queries/registrations.graphql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+query registrations($taalhuisId: String!) {
+    registrations(taalhuisId: $taalhuisId) {
+        id
+        dateCreated
+        status
+        givenName
+        additionalName
+        familyName
+        registrar {
+            organisationName
```
Comment (Grkmk) [inline client/src/graphql/queries/registrations.graphql:10]: i think this is missing the id

**Review by Grkmk — CHANGES_REQUESTED** [review]

File: client/src/graphql/queries/registrations.graphql (hunk 1 above)
Comment (RickWoltheus) [reply client/src/graphql/queries/registrations.graphql:10]: image
there is no id available right now

Comment (Grkmk) [reply client/src/graphql/queries/registrations.graphql:10]: sounds good. can you leave a `TODO` in the code so that we wont miss it?

**Review by Grkmk — DISMISSED** [review]

File: client/src/graphql/queries/registrations.graphql (hunk 1 above)
Comment (RickWoltheus) [reply client/src/graphql/queries/registrations.graphql]: done, i need an approve again though

**Review by Grkmk — APPROVED** [review]

**Pull request merged**